src/dotnet/UI.Blazor/Components/Toast/ErrorToast.razor

```diff
@@ -25,10 +25,13 @@
         </div>
         <div class="c-message">@ChildContent</div>
     </div>
-    <ButtonX
-        Class="btn-transparent btn-xs"
-        Click="@(_ => Dismiss())">
-    </ButtonX>
+    <TimerCloseButton
+        Class="btn-sm"
+        CircleClass="btn-sm"
```

Comment:
Do we need a separate class for circle? May be better to solve it on css level?
button.btn-sm circle.circle {
  // svg style
}